packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/process/schema.ts

```diff
@@ -70,7 +64,6 @@ export const QueryByParameterSchema = z.object({
             _root: z.string(),
           })
         ).optional(),
-        semanticsText: z.literal("AssignedProvider.id"),
```

Comment:
Is there no use for this at all that would warrant us making it optional?

Reply:
We dont currently use it at all and different implementers dont use the cannonical values in the spec. If we make it optional, but the value is defined and wrong, then we throw a schema error which we dont want